desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -125,7 +128,15 @@ module.exports = class ArtworkAuctionView extends Backbone.View
       variables: id: AUCTION.artwork_id, sale_id: AUCTION.id
       req: user: CURRENT_USER
     .then (data) =>
+      old_data = @data
       @data = extend data, user: CURRENT_USER?
-      @render()
+      
+      @render() unless isEqual(old_data, data)
```

Comment:
Apparently `isEqual` does a deep compare, which is handy here. It's still not great that we re-render a component that the user might be in the middle of interacting with. But comparing the data driving the render seemed like the easiest partial fix.